pg_lakehouse/README.md

```diff
@@ -0,0 +1,299 @@
+<h1 align="center">
+  <img src="../docs/logo/pg_lakehouse.svg" alt="pg_lakehouse" width="500px">
+<br>
+</h1>
+
+## Overview
+
+`pg_lakehouse` is an extension that transforms Postgres into an analytical query engine over object stores like S3 and table formats like Apache Iceberg. Queries are pushed down to [Apache DataFusion](https://github.com/apache/datafusion), which delivers excellent analytical performance. Combinations of the following object stores, table formats, and file formats are supported.
```

Comment:
@rebasedming We might want to write something here about being able to write to S3 from Postgres too. It seems to be pretty valuable